hmi_rosbridge/message[] VehicleMessage

